src/registrar/tests/test_models_domain.py

```diff
@@ -2320,7 +2344,7 @@ def setUp(self):
         self.domain, _ = Domain.objects.get_or_create(name="fake.gov", state=Domain.State.READY)
         # creation_date returned from mockDataInfoDomain with creation date:
         # cr_date=datetime.datetime(2023, 5, 25, 19, 45, 35)
-        self.creation_date = make_aware(datetime.datetime(2023, 5, 25, 19, 45, 35))
+        self.creation_date = make_aware(datetime(2023, 5, 25, 19, 45, 35))
```

Comment:
did not know this was a thing in Django! learned something new today, thank you 🎉